packages/angular-workspace/angular.json

```diff
@@ -237,7 +246,7 @@
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
             "main": "example-client-app/src/test.ts",
-            "polyfills": "example-client-app/src/polyfills.ts",
+            "polyfills": ["zone.js"],
```

Comment:
polyfills is already configured in the top-level "example-client-app" configuration.

Is it intentional that it is redefined differently here (the project level one includes the localize/init)? Can we opt-out of overriding it here? If it needs to be defined here does it need to be different?

Reply:
The other one isn't "top-level", but rather for the build, whereas this is for testing, so I think we need it in both places. It doesn't seem to matter that the tests aren't importing `@angular/localization/init` (they pass), but we might as well keep them consistent.